accepted/0000-link-packages-to-source-and-build.md

```diff
@@ -0,0 +1,514 @@
+# Link npm packages to the originating source code repository and build
+
+**Authors**: @feelepxyz @kommendorkapten @trevrosen
+
+## Summary
+Link public npm packages to the source code repository and build it originated from, making it harder to execute package hijacking attacks where a malicious version of an existing open source package gets uploaded to the registry.
+
+Check out the [glossary](#glossary) for an overview of the different tools, terms and techniques covered in this RFC document.
+
+## Motivation
+So far our supply chain security offerings have focused on remediation - patching known vulnerabilities in dependencies (e.g. `npm audit`). The majority of vulnerabilities are accidental, so that was the right place to start. Now we need to turn our attention to deliberate supply chain attacks. In 2021, we saw these types of attacks increase by a terrifying [650%](https://www.sonatype.com/resources/state-of-the-software-supply-chain-2021).
+
+In November of last year alone, we saw three supply chain attacks against popular npm packages ([UA-Parser-JS](https://www.esentire.com/security-advisories/npm-library-supply-chain-attack), [COA](https://www.esentire.com/security-advisories/coa-npm-supply-chain-attack), and [RC](https://www.esentire.com/security-advisories/rc-npm-supply-chain-attack)), each with millions of downloads per month.
```

Comment:
```suggestion
In November of last year alone, we saw three supply chain attacks against popular npm packages ([UAParser.js](https://www.esentire.com/security-advisories/npm-library-supply-chain-attack), [Command-Option-Argument](https://www.esentire.com/security-advisories/coa-npm-supply-chain-attack), and [
](https://www.esentire.com/security-advisories/rc-npm-supply-chain-attack)), each with millions of downloads per month.
```

We should try to get the names of the projects as they self identify, rather than as what a third-party site (that also gets npm's name wrong) uses. These changes pulled with preference from the project's GitHub repo descriptions.